submissions/nauvian-perspective-bentham/nauvian-perspective-bentham.md

```diff
@@ -0,0 +1,50 @@
+
+Over the course of 2014, Bentham began a few different Factorio series. In August 2014, he began his popular and most favourite [Factorio Railworld](https://www.youtube.com/watch?v=aw9zgzrF2DY&list=PLOx2-nvzE_ZI69s1psHkAdeH2_uNmP9rg&index=1) series and the [Factorio No Crafting Challenge](https://www.youtube.com/watch?v=FTszEGCARW4&list=PLOx2-nvzE_ZKIrTTzJ5etReO2iFeo_WIp) series. Each video in the Railworld series attracted tens of thousands of views which grew his channel from a few hundred subscribers to many thousands.
+
+![Midpoint Station from Ep#63](./Midpoint-Station.png)
+
+It is sometimes thought that Bentham was the first person who properly coined the idea of a "main bus", although at the time he called it the "tree method", as the bus was the trunk with production lines on the sides as the branches. [He said in the interview](./Tree-Method.mp3) that in the forums, the "main bus" phrase started cropping up, and it took him a while to realise that it was the same thing as his tree method. Of course it's likely that the main bus is just the natural evolution of many Factorio players that wish to leave the spaghetti playstyle. 
+
+> I just had an epiphany while building a factory and being like "urgh everything's so messy and all over the place, how can I make this scale", and I had a think about it and I'm like "oh! oh! you just put everything down the middle and have the assemblers come out the side, this is genuis!"
+
+If you are a long-time Factorio fan, you may have heard of him from his very popular [Factorio Rail Tutorials](https://youtu.be/XSGYSbEPpbM) which provided a much-needed handy guide to understanding and using trains and signals. Rails were very annoying to build back then, having the curved rail and straight rail, and where the rail signal icon had 3 lights but actually only had 2.
+
+Later, in 2015, he began the [Living with Biters](https://www.youtube.com/watch?v=83KSIH4dJg8&list=PLOx2-nvzE_ZJXS1sQDUzWtvBLn9ZPyUXk) series. The idea of living amongst biter bases on peaceful mode is fascinating, as it poses multiple challenges you wouldn’t get in any other normal vanilla campaign. This base is so interesting that stringweasel plans to put it into his fantastic [Hall of Fame mod](https://mods.factorio.com/mod/HallOfFame).
+
+![Living with Biters Ep#22 Thumbnail](./Living-With-Biters-Base.jpeg)
```

Comment:
Again, the low quality is unfortunate. It might be possible to find a better capture where the image is zoomed in and there is assmeblers, bases, and biters visible. Although with a quick look it seemed to be hard to find a nice shot. I'll try to have a look later this week.

Reply:
However, all the images do need captions. We have some funky formatting for it, you can just write it in italics below the image. Conor will fix it later. 

This one could for example say:

> Bentham trying to get steel underneath biter nests using underground belts in his Living With Biters series.

Reply:
Alright. Well, for now, until we work out exactly what images we'll include, I think it's best to leave it, as of course different captions will have to be used.